tests/capture/autograph/test_autograph.py

```diff
@@ -110,13 +110,12 @@ def test_transform_on_lambda(self):
         assert new_fn(1.23) == 2.46
         assert "inner_factory.<locals>.<lambda>" in str(new_fn)
 
-    @pytest.mark.parametrize("autograph", [True, False])
-    def test_transform_on_qnode(self, autograph):
+    def test_transform_on_qnode(self):
```

Comment:
Tests like these used to run with autograph enabled and disabled, and now we only run them once with whatever the default is wrt. autograph. Is this intended?